### PR TITLE
Added Aletheia Data

### PR DESCRIPTION
**Pull request by aletheiadata**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,6 +186,7 @@ Table of Contents
   * [IP.City](https://ip.city) — 100 free IP geolocation requests per day
   * [Abstract API](https://www.abstractapi.com) — API suite for a variety of use cases including IP geolocation, gender detection or even email validation.
   * [algorithmia.com](https://algorithmia.com/) — Host algorithms for free. Includes free monthly allowance for running algorithms. Now with CLI support.
+  * [Aletheia Data](https://aletheiadata.org/) — Open Source Open API that utilize Web3 Technologies to democratize public information and facilitate access to it.
```
Comment (ripienaar) [inline README.md:189]: I am trying to figure out what this is about - but I cannot find anywhere on the docs site a list of data thats available or anything really, can you give me a pointer to somewhere I can see what data is available in order to figure out if its on topic for this list?

**Review by DebiFaye — APPROVED** [review]

**Comment by aletheiadata** [comment]
Hi @ripienaar ,

sure. So, I use the Rapid API to distribute my services. On this link you can find our Basic API with is basically an indexer to query our files. Our files are decentralized and they live in the IPFS ecosystem. To query those files is enough to call our service with the correct CID (you can get the CID from our (dashboard)[https://app.aletheiadata.org] - you'll need metamask to login). Here is an example: 

CID: bafybeid2snlboow3e2ukz2lbw7a5yv5ach642s7cd43eyavyiwqs7txkpe
Enter this CID in https://rapidapi.com/aletheia-data-aletheia-data-default/api/aletheia2 under V1/SEARCH - IPFS endpoint (V1)

`var axios = require("axios").default;

var options = {
  method: 'GET',
  url: 'https://aletheia2.p.rapidapi.com/v1/_search/ipfs/csv/bafybeid2snlboow3e2ukz2lbw7a5yv5ach642s7cd43eyavyiwqs7txkpe',
  params: {
    start: '0',
    limit: '10',
    info: 'true',
    fields: 'fields1,field2',
    value: 'search in field(s)',
    order: 'id:asc'
  },
  headers: {
    'x-rapidapi-host': 'aletheia2.p.rapidapi.com',
    'x-rapidapi-key': '4606bf9087msh30f2baceb3f9ceap1fac62jsnd2ba6b7c2f7d'
  }
};

axios.request(options).then(function (response) {
	console.log(response.data);
}).catch(function (error) {
	console.error(error);
});`

Basically, any file becomes accessible, downloadable, reliable at any time. My intent is to provide permanent storage for public information and give access to it in a public, open, and transparent way. 

Soy un Dominicano que con pasión y poco dinero :-p estoy construyendo poco a poco una herramienta para que nuestros desarrolladores no tengan que pasar lucha buscando información pública y puedan tener una fuente confiable de información publica / privada. 

Si tienes mas pregunta, con gusto te respondo 🤙🏽

File: README.md (hunk 1 above)
Comment (aletheiadata) [reply README.md:189]: Hi @ripienaar ,

sure. So, I use the Rapid API to distribute my services. On this [link](https://rapidapi.com/aletheia-data-aletheia-data-default/api/aletheia2) you can find our Basic API with is basically an indexer to query our files. Our files are decentralized and they live in the IPFS ecosystem. To query those files is enough to call our service with the correct CID (you can get the CID from our (dashboard)[https://app.aletheiadata.org] - you'll need metamask to login). Here is an example: 

CID: bafybeid2snlboow3e2ukz2lbw7a5yv5ach642s7cd43eyavyiwqs7txkpe
Enter this CID in https://rapidapi.com/aletheia-data-aletheia-data-default/api/aletheia2 under V1/SEARCH - IPFS endpoint (V1)

`var axios = require("axios").default;

var options = {
  method: 'GET',
  url: 'https://aletheia2.p.rapidapi.com/v1/_search/ipfs/csv/bafybeid2snlboow3e2ukz2lbw7a5yv5ach642s7cd43eyavyiwqs7txkpe',
  params: {
    start: '0',
    limit: '10',
    info: 'true',
    fields: 'fields1,field2',
    value: 'search in field(s)',
    order: 'id:asc'
  },
  headers: {
    'x-rapidapi-host': 'aletheia2.p.rapidapi.com',
    'x-rapidapi-key': '4606bf9087msh30f2baceb3f9ceap1fac62jsnd2ba6b7c2f7d'
  }
};

axios.request(options).then(function (response) {
	console.log(response.data);
}).catch(function (error) {
	console.error(error);
});`

Basically, any file becomes accessible, downloadable, reliable at any time. My intent is to provide permanent storage for public information and give access to it in a public, open, and transparent way. 

Please take into consideration this is still under development.

Soy un Dominicano que con pasión y poco dinero :-p estoy construyendo poco a poco una herramienta para que nuestros desarrolladores no tengan que pasar lucha buscando información pública y puedan tener una fuente confiable de información publica / privada. 

Si tienes mas pregunta, con gusto te respondo 🤙🏽

Comment (aletheiadata) [reply README.md:189]: Think it this way, it's like a public directory. Or a library with public files you can access to. But it's ok, I understand, don't worries.

Comment (ripienaar) [reply README.md:189]: Yes. A library with no catalogue :)

you are telling me to go through all the rows of books myself instead of giving me a easy way to see what subject matters you cover - art or science library. 

Anyway, to maintain the list I need to evaluate each entry. I am not signing up for things to do that

thank you - if you do eventually make some index or sometning feel free to come again :)

Comment (aletheiadata) [reply README.md:189]: ok, i'm not sure what you're asking me to do here, but if you expect me to do a list of all my files/records (currently i have more than 10,000 records), then i don't think is gonna happen. Thanks a lot anyway
